src/plugins/mer/mervirtualmachinesettingswidget.h

```diff
@@ -47,7 +52,7 @@ class MerVirtualMachineSettingsWidget : public QWidget
     void setMemorySizeMb(int sizeMb);
     void setCpuCount(int count);
     void setStorageSizeMb(int storageSizeMb);
-    void setVmOff(bool vmOff);
+    void setVmOff(Sfdk::VirtualMachine * const virtualMachine, bool vmOff);
```

Comment:
This would not be changed and instead we would add `void setVmFeatures(...)` likely as the very first setter.

Reply:
The signature isn't changed, but the implementation still needs some changes to make sure that the controls aren't enabled